packages/snaps-cli/src/cmds/build/bundleUtils.ts

```diff
@@ -128,3 +129,56 @@ export function postProcess(
 
   return processedString;
 }
+
+/**
+ * Processes dependencies and updates argv with an options object
+ * @param argv
+ */
+
+export function processDependencies(argv: YargsArgs) {
+  const { transpiledDeps, transpilationMode } = argv;
+  const babelifyOptions: Record<string, any> = {};
+  if (transpilationMode === TranspilationModes.localAndDeps) {
+    const regexpStr = getDependencyRegExp(transpiledDeps as string[]);
```

Comment:
This is no longer a string:
```suggestion
    const regexpArray = getDependencyRegExp(transpiledDeps as string[]);
```

Reply:
Although, it might be better if it were, and the array wrapping happens inside `processDependencies` when we add the array to `babelifyOptions`.